packages/client/hmi-client/src/services/workflow.ts

```diff
@@ -276,8 +276,8 @@ export const updateWorkflow = async (workflow: Workflow) => {
 };
 
 // Get
-export const getWorkflow = async (id: string) => {
-	const response = await API.get(`/workflows/${id}`);
+export const getWorkflow = async (id: string, projectId?: string) => {
+	const response = await API.get(`/workflows/${id}`, { params: { 'project-id': projectId } });
```

Comment:
Why do we need to pass in a projectId?

Reply:
We are already passing active projectId for all requests that's handled by axios interceptor but when the request is called where you are out side of the project, projectId isn't available. For notification panel in the  main root page, active projectId isn't available and we are providing it manually for this method.

Reply:
I see, so this is a hack to get the ID int the URL in order to trigger the interceptor, which at the notification component you may or may not have the active-project? 

Then why not just wait for the active project to be available before showing the notification component? 

At least we need some documentations/comments if doing a hack, because it is not clear why project_id is passed in, as it does not match the workflow-controller's API signatures.

Reply:
Added comments to those methods.